Offline-Calendar/src/main/AndroidManifest.xml

```diff
@@ -45,7 +46,8 @@
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
-        android:theme="@style/Theme.AppCompat.Light.NoActionBar">
+        android:theme="@style/Theme.AppCompat.NoActionBar">
```

Comment:
Is there a way to stop the flickering when starting an app with one theme set in the manifest and another one set via java?